cmd/createcluster.go

```diff
@@ -139,6 +139,31 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		return errors.New("number of --keymanager-addresses do not match --keymanager-auth-tokens. Please fix configuration flags")
 	}
 
+	var secrets []tbls.PrivateKey
+
+	switch conf.SplitKeys {
+	case true:
```

Comment:
what about a good old `if else` rather?

Reply:
the effect is the same, i can replace it if you want

Reply:
done